tests/RobotFramework/tests/cumulocity/configuration/configuration_with_file_transfer_https.robot

```diff
@@ -79,18 +87,27 @@ Get Configuration Should Succeed
 Get Configuration Should Fail
     [Arguments]    ${failure_reason}    ${device}    ${external_id}
     Cumulocity.Set Device    ${external_id}
-    ${operation}=    Cumulocity.Get Configuration    CONFIG1
+    ${operation}=    Cumulocity.Get Configuration    tedge-configuration-plugin
```

Comment:
@jarhodes314 What was the main reason for using `tedge-configuration-plugin` over the `CONFIG1`? Was it just easier to use because former is always there?